Backend/app/therapy_sessions/serializers.py

```
@@ -0,0 +1,738 @@
+    def validate_therapist_id(self, value):
+        """Validate that therapist exists and patient is connected to them"""
+        try:
+            therapist = User.objects.get(id=value, user_type='therapist')
+            patient = self.context['request'].user
+            
+            # Check if patient is connected to this therapist
+            if not hasattr(patient, 'patient_profile') or not patient.patient_profile.therapist or patient.patient_profile.therapist.user != therapist:
+                raise serializers.ValidationError("You are not connected to this therapist.")
+            return value
+        except User.DoesNotExist:
+            raise serializers.ValidationError("Therapist not found.")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)